.github/workflows/shopify-cli.yml

```diff
@@ -85,12 +85,14 @@ jobs:
         run: yarn install --ignore-engines
       - name: Build
         run: npx nx ${{ github.event.inputs.nxLevel || 'affected' }} --target=build --skip-nx-cache
-      - name: Test
-        run: npx nx ${{ github.event.inputs.nxLevel || 'affected' }} --target=test --skip-nx-cache
       - name: Lint
         run: npx nx ${{ github.event.inputs.nxLevel || 'affected' }} --target=lint --skip-nx-cache
       - name: Type-check
         run: npx nx ${{ github.event.inputs.nxLevel || 'affected' }} --target=tsc --skip-nx-cache
+      - name: Unit tests
+        run: npx nx ${{ github.event.inputs.nxLevel || 'affected' }} --target=test --exclude=features --skip-nx-cache
+      - name: Acceptance tests
+        run: npx nx ${{ github.event.inputs.nxLevel || 'run' }} features:test --skip-nx-cache
```

Comment:
no need for the github variable here, since this is to execute only feature:test, the variable in `github.event.inputs.nxLevel` will be always incorrect.

Let's change it to `npx nx run features:test --skip-nx-cache`